src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -253,7 +253,7 @@ private function getSerializerContext(string $resourceClass, string $contextType
             $request = $this->requestStack->getCurrentRequest();
         }
 
-        if (null !== $this->serializerContextBuilder && null !== $request) {
+        if (null !== $this->serializerContextBuilder && null !== $request && 'api_graphql_entrypoint' !== $request->attributes->get('_route')) {
```

Comment:
I suggest to check for the `application/graphql` MIME type instead. It will be more reliable.

Reply:
Forget about that, content-type can also be `application/json`, so it's a bad idea. I'm still not a found of checking the route name (it creates a hard coupling with the Symfony Routing component).

Maybe can you just set a new `_graphql` attributes to true in the action, and check it here. It will be easier for end users to hook into this.